soc/xtensa/intel_adsp/cavs_v15/include/soc/memory.h

```diff
@@ -18,8 +18,8 @@
 #define L2_SRAM_SIZE (DT_REG_SIZE(DT_NODELABEL(sram0)))
 
 #ifdef CONFIG_BOOTLOADER_MCUBOOT
```

Comment:
hmm, why do we have mcuboot here in the first place, we never use mcuboot with those platforms, I would just remove the whole thing.

Reply:
@nashif done